Exiled.API/Features/Scp330.cs

```diff
@@ -0,0 +1,162 @@
+// -----------------------------------------------------------------------
+// <copyright file="Scp330.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    using System.Collections.Generic;
+
+    using InventorySystem;
+    using InventorySystem.Items.Usables.Scp330;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// A Wrapper class for <see cref="Scp330Bag"/>.
+    /// </summary>
+    public class Scp330
```

Comment:
Item wrappers should inherit the Features/Items/Item class, or the most appropriate subclass (in this case, Usable)